client/assets/stylesheets/common/_vuetify.scss

```diff
@@ -22,3 +22,7 @@ a:hover, a:active, a:visited, a:focus {
 hr.v-divider--vertical {
   margin: 0;
 }
+
+.v-input--selection-controls__input input {
+  position: absolute;
```

Comment:
This does not seem to do anything to affect the styling since `Veutify` already applies `absolute` positioning in one of its stylings.

Reply:
If fixes the misalignment of the checkboxes inside the selection section on built version of the app. Vuetify does apply the `absolute` positioning, but bootstrap overrides it.

Reply:
I see it now. In that case, write a comment with a TODO to remove the code when Bootstrap is removed.

Reply:
I didn't notice the first time I read the file but there is a comment at the top of the file that states that the purpose of all the styling inside it is for fixing Bootstrap conflicts. So there is no need for the comment I mentioned. Sorry about that.

Reply:
Oh right, should have seen it myself. Removing the comment